cps/src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java

```diff
@@ -329,6 +332,34 @@ private static void describeType(DescribableHelper.ParameterType type, PrintWrit
         }
     }
 
+    @Restricted(NoExternalUse.class)
+    public static final String GDSL_URL = ACTION_URL + "/gdsl";
+
+    @Restricted(DoNotUse.class)
+    public void doGdsl(StaplerRequest req, StaplerResponse rsp) throws Exception {
+        rsp.setContentType("text/plain;charset=UTF-8");
+        req.getView(this,"gdsl.groovy").forward(req, rsp);
```

Comment:
Can delete this method. Stapler should find the view anyway. (use `st:header` as needed)